ci-conda.Dockerfile

```diff
@@ -1,12 +1,21 @@
 ARG CUDA_VER=notset
 ARG LINUX_VER=notset
 ARG PYTHON_VER=notset
+ARG YQ_VER
+ARG AWS_CLI_VER
+ARG SCCACHE_VER
+
+FROM mikefarah/yq:${YQ_VER} as yq
+
+FROM amazon/aws-cli:${AWS_CLI_VER} as aws-cli
+
```

Comment:
only `ARG`s that are used in the `FROM` directives need to be specified at the top.

the remaining `ARG`s should come after the `FROM` statements, otherwise they won't be recognized throughout the rest of the Dockerfile.

These docs outline this behavior: https://docs.docker.com/reference/dockerfile/#understand-how-arg-and-from-interact.